wbooks/src/app/screens/booklist/components/BookItem/index.js

```diff
@@ -4,18 +4,20 @@ import styles from './styles';
 
 class BookItem extends Component {
 
-	render() {
-		return (
+  render() {
+    const { urlPath, title, writer } = this.props;
+    
+    return (
       <View style={styles.container}>
         <Image style={styles.image}
-        source={{uri: 'http://wolox-training.s3.amazonaws.com/uploads/6942334-M.jpg'}} />
+        source={{uri: urlPath}} />
```

Comment:
change the prop `urlPath` to `uri` to allow this to happen: 
`source={{uri}}`

Reply:
great trick! done